augly/audio/functional.py

```diff
@@ -618,24 +619,24 @@ def invert_channels(
 
 
 def loop(
-    audio: Union[str, np.ndarray],
+    audio: Union[str, cp.ndarray],
```

Comment:
We should still take in the audio as an np array, and then convert it to a cupy array in the augmentation. We don't want to change the API for users, we just want the augmentation to speed up.

Reply:
@zpapakipos If they pass in a numpy array it will still accept it as a cupy array when I was testing it. Should I change it anyway?